calculator.js

```diff
@@ -1,5 +1,65 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const prompt = require('prompt');
+
+// Built off of the Ada curriculum hints for using prompt
+const calculateUserInput = function(error, promptInput) {
+  console.log(`Calculating: ${promptInput.num1} ${promptInput.operation} ${promptInput.num2}`);
+
+  switch (promptInput.operation) {
+    case "+":
+    case "add":
+      console.log(`Result: ${promptInput.num1 + promptInput.num2}`);
+      break;
+    case "-":
+    case "subtract":
+      console.log(`Result: ${promptInput.num1 - promptInput.num2}`);
+      break;
+    case "*":
+    case "multiply":
+      console.log(`Result: ${promptInput.num1 * promptInput.num2}`);
+      break;
+    case "/":
+    case "divide":
+      if (promptInput.num2 === 0) return console.log("ERROR: Cannot divide by 0.");
+      console.log(`Result: ${promptInput.num1 / promptInput.num2}`);
+      break;
+    case "^":
+    case "exponential":
+      console.log(`Result: ${Math.pow(promptInput.num1, promptInput.num2)}`);
+      break;
+    case "%":
+    case "mod":
+      console.log(`Result: ${promptInput.num1 % promptInput.num2}`);
+      break;
+  }
+} 
```

Comment:
This is not very intuitive but this is still a variable definition so a `;` technically goes at the end.
JS is super flexible so it works hard to add them for you when it's pretty sure you forgot one, but our best practice at Ada is to try to add them everywhere they go rather than relying on JS to do that for us.